### PR TITLE
DOC Users-Guide documentation added

### PR DESCRIPTION
**Pull request by jmmovilla**
@hmunfru @flopezag @jframos

### CODE REVIEW

File: doc/user_guide.rst [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+	-X POST "https://saggita.lab.fi-ware.org:8443/sdc/rest/catalog/product/{product-name}"
+
+with the following payload
+
+.. code::
+
+	<product>
+ 		<name>{product-name}</name>
+ 		<description>Description</description>
+	</product>
```
Comment (jframos) [inline doc/user_guide.rst]: Perhaps we can show in that code example how to create a product with metadata and attribute values

File: doc/user_guide.rst [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+
+with the following payload
+
+.. code::
+
+	<productReleaseDto>
+		<productName>{product-name}</productName>
+		<version>{version}</version>
+		<productDescription>description of {product-name}-{version}/productDescription>
+	</productReleaseDto>
```
Comment (jframos) [inline doc/user_guide.rst]: "productName" and "productDescription" should be sent in this request?

File: doc/user_guide.rst [hunk 3]
```diff
@@ -0,0 +1,486 @@
...
+Next we detail some operations that can be done in the catalogue management api
+
+Product API
+-----------
+
+**Get the Product List from the catalogue**
+
+.. code::
+
+    $ curl -v -H "Content-Type: application/json" -H "Accept: application/json" -H
```
Comment (jframos) [inline doc/user_guide.rst]: The example response is in XML format. Accept header should be "application/xml".
(The same for the rest of curl requests in this document)

File: doc/user_guide.rst [hunk 4]
```diff
@@ -0,0 +1,486 @@
...
+			<description>mysql installator</description>
+ 		</metadatas>
+ 	</product>
+
+
+**Add a New Product to the catalogue**
+
+.. code::
+
+    $ curl -v -H "Content-Type: application/json" -H "Accept: application/json" -H
```
Comment (jframos) [inline doc/user_guide.rst]: The example body is in XML format. Content-Type header should be "application/xml".
(The same for the rest of curl requests in this document)

File: doc/user_guide.rst [hunk 5]
```diff
@@ -0,0 +1,486 @@
...
+    "X-Auth-Token: 756cfb31e062216544215f54447e2716" -H "Tenant-Id: your-tenant-id" 
+	-X POST "http://saggita.lab.fi-ware.org:8080/sdc/rest/vdc/{your-tenant-id}/productInstance"
+
+where {your-tenant-id} is the tenant-id in this guide. The payload of this request can be as follows:
+
+.. code::
+	
+	<productInstanceDto>
```
Comment (jframos) [inline doc/user_guide.rst]: We can show in this request how to configure installation attributes

**Comment by jframos** [comment]
I can't see documented the operation to remove a node from Chef-Server / PuppetMaster
```
/sdc/rest/vdc/{tenant-id}/chefClient/{hostname}
```

File: doc/user_guide.rst (hunk 2 above)
Comment (jmmovilla) [inline doc/user_guide.rst]: productName and productDescription are not compulsory

**Comment by hmunfru** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Pull request merged**